examples/deployment-chain/simple/.pipe.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: pipecd.dev/v1beta1
+kind: KubernetesApp
+spec:
+  input:
+    manifests:
+      - deployment.yaml
+      - service.yaml
+    kubectlVersion: 1.18.0
+    namespace: khanhtc-dev
+  postSync:
+    chain:
+      applications:
+        - name: simple-b
+        - name: simple-c
```

Comment:
We should prepare those apps as well since all examples should be expected to be able to run on the play-ground environment in the near future.

Reply:
Just want to confirm one thing, this one is for examples so I think if we also add manifests for those applications (which be triggered once this first application is triggered) here, the example may too complicated and contain something that users don't need to really care about, so I guess it's simpler to just add the manifest for the first application in chain. I don't have a strong opinion on this, but how do you think about that?

Reply:
I feel you. But IMO, `runnable` examples will be helpful for our users when they try to use them or when they explore them in our playground environment to see how they are actually running.

How about reusing the examples in the existing directory such as`kubernetes`, `terraform` for that list?

Reply:
Nice idea, will use those existing applications for this example 👍